docs/messaging.md

```diff
@@ -46,7 +46,7 @@ notifications count and a link to the notifications page.
 On the notifications page, users can view, follow calls to action on, and
 dismiss their notifications. They can also click the "Dismissed" tab to view
 notifications they've previously dismissed. High priority notifications
-float to the top of the lists.
+(DEPRECATED) float to the top of the lists.
```

Comment:
Expected to see the deprecated message at the end of the bulleted list item like the header of this section: 
`- float to the top of the lists (DEPRECATED)`

Could go either way, but would do one or the other just for consistency's sake. Messaging is very clear, and I would understand this as a new developer coming into the project.

Reply:
That's not a bulleted list item, though GitHub's diff treatment does make it look like a Markdown list item.

It's a sentence, changing 

> High priority notifications float to the top of the lists.

to 

> High priority notifications (DEPRECATED) float to the top of the lists.

So the DEPRECATED is placed close to the aspect that's deprecated.

I figure this is probably good enough for now, this may be short lived docs anyway if @thevoiceofzeke 's adding the banner message UI that conceptually replaces "priority" notifications continues to go well. :)